ocean_lib/enforce_typing_shim.py

```diff
@@ -15,6 +16,15 @@ def setup_enforce_typing_shim():
 
 
 def enforce_types_shim(func):
-    if not ConfigProvider.get_config()["util"].getboolean("typecheck"):
+    try:
+        c = ConfigProvider.get_config()
+    except AssertionError:  # handle if ConfigProvider.set_config() not done yet
+        c = config_defaults
+
+    val = c["util"][NAME_TYPECHECK]
+    typecheck = distutils.util.strtobool(val)
+
+    if typecheck:
+        return enforce_types(func)
+    else:
```

Comment:
we don't need the else branch. If the return inside the "if" branch is activated, there is no way of getting inside the "else" branch. (you can just remove the "else" and unindent `return func`)

Reply:
I'm fine with this suggestion, I don't know the most pythonic way. Changed.